adaptive_breakpoints/example/test/widget_test.dart

```diff
@@ -0,0 +1,12 @@
+// This is a basic Flutter widget test.
```

Comment:
Do you actually need to include this? Looks like the test does nothing

If you do need it, then please add the copyright header

Reply:
Done